Source/bmalloc/bmalloc/BPlatform.h

```diff
@@ -392,7 +392,7 @@
 #define BENABLE_MALLOC_GOOD_SIZE 0
 #endif
 
-#if BUSE(LIBPAS) && BOS(DARWIN) && BCPU(ARM64)
+#if !defined(BUSE_TZONE) && BUSE(LIBPAS) && BOS(DARWIN) && BCPU(ARM64)
 #define BUSE_TZONE 0
 #else
 #define BUSE_TZONE 0
```

Comment:
Does this work?  It looks like if `defined(BUSE_TZONE)`, it will fall into the `#else` case which `#define BUSE_TZONE 0`

Reply:
Fixed.